### ISSUE TITLE
Trained Models attention only on recent data

### ISSUE THREAD

**Issue by Clickative** [jdb78/pytorch-forecasting#165]
- PyTorch-Forecasting version: 0.6.0
- PyTorch version: 1.6
- Python version: 3.6
- Operating System: Linux

I made a model to predict some stock market features but the results seem to be very weird. I used the "Demand forecasting with the Temporal Fusion Transformer" example as a base but my model seems to be off, despite having a larger size, more training, more data etc.

    learning_rate=0.013489628825916528, #I got the learning rate by tuner.
    hidden_size=64,
    attention_head_size=16,
    dropout=0.1,
    hidden_continuous_size=16,
    output_size=7,
    loss=QuantileLoss(),
    log_interval=10,
    reduce_on_plateau_patience=4,

Here are some prediction examples:
![download (9)](https://user-images.githubusercontent.com/35162301/99843084-f6c31180-2b96-11eb-8342-9bc42c276142.png)
![download (8)](https://user-images.githubusercontent.com/35162301/99843088-f7f43e80-2b96-11eb-8dc3-ae5e8221ec49.png)
![download (7)](https://user-images.githubusercontent.com/35162301/99843089-f88cd500-2b96-11eb-9153-ec1a5b227649.png)

Since I'm using colab for training, I did run around 100 epochs, but there were no good. My dataset has 488468 rows, and 6 time_varying_unknown_reals. I really don't understand why at this point the network is misbehaving.

Quick Note:
I also calculated the "Actuals vs predictions by variables" and the results in that part seem to be a little more promising but ended up confusing me even more.
![download (12)](https://user-images.githubusercontent.com/35162301/99843615-dc3d6800-2b97-11eb-80f1-3175fc433f36.png)
![download (11)](https://user-images.githubusercontent.com/35162301/99843623-de072b80-2b97-11eb-9fc4-f91990e96456.png)
![download (10)](https://user-images.githubusercontent.com/35162301/99843624-de9fc200-2b97-11eb-9a0a-f30cc9500009.png)

**Comment by jdb78** [comment]
Could you provide the dataset definition? Stock market prices are notoriously difficult to predict. If you want to generate alpha, I believe you need more than just the stock prices.
